.github/PULL_REQUEST_TEMPLATE/release-pull-request.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Pull request - release
+about: Provide specifics about a pull request for releases
+title: 'PR - release: ____(provide a descriptive name)'
+labels: release
+assignees: ''
+
+---
+<!-- IMPORTANT: Please do not create a Pull Request without creating an issue first. -->
+
+<!--Fields in **bold** are REQUIRED, fields in *italics* are OPTIONAL -->
+
+**Description**
+<!-- A clear and concise description of the updates contained in this release. We usually use a list. -->
+
+Improvements include:
+- A #issue
+- B #issue
+- C #issue
+- n  
```

Comment:
Not sure what you're trying to do here. For releases, we want to auto close issues.